### PR TITLE
Updating README with service catalog and other small updates.

### PR DESCRIPTION
**Pull request by mmartin24**
Updates done:

- Added mention to Epinio's catalog services.
- Updated link to Epinio binaries to v1.2.0
- Updated `BP_PHP_VERSION` to `8.0.x`
- Updated create cluster command from `k3d cluster create epinio -p 80:80 -p 443:443` to `k3d cluster create epinio` 

Preview:
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,30 @@ Wordpress needs a database to work. After visiting the route of your deployed ap
 
 You can install a MySQL database on your cluster or use an external one. One option is using a helm chart like this one: https://bitnami.com/stack/mysql/helm
 
+Also you may use the ones already available on our catalog services:
```
Comment (nunix) [inline README.md]:
```suggestion
Alternatively, you may use the ones already available on our `service catalog`:
```

File: README.md [hunk 2]
```diff
@@ -47,6 +47,30 @@ Wordpress needs a database to work. After visiting the route of your deployed ap
...
+#| rabbitmq-dev   | 2022-09-22 12:45:05 +0200 CEST | 3.9.17  | A RabbitMQ service that can be |
+#|                |                                |         | used during development        |
+#| mysql-dev      | 2022-09-22 12:45:05 +0200 CEST | 8.0.29  | A MySQL service that can be    |
+#|                |                                |         | used during development        |
+#| mongodb-dev    | 2022-09-22 12:45:05 +0200 CEST | 6.0.1   | A MongoDB service that can be  |
+#|                |                                |         | used during development        |
+#| redis-dev      | 2022-09-22 12:45:05 +0200 CEST | 6.2.7   | A Redis service that can be    |
+#|                |                                |         | used during development        |
+```
+More info about our Service Catalog here: https://docs.epinio.io/references/customization/catalog
```
Comment (nunix) [inline README.md]:
```suggestion
More information about Epinio Service Catalog is available in [the docs](https://docs.epinio.io/references/customization/catalog).
```

File: README.md [hunk 3]
```diff
@@ -47,6 +47,30 @@ Wordpress needs a database to work. After visiting the route of your deployed ap
...
+
+For this example, we proceed with `mysql-dev`:
```
Comment (nunix) [inline README.md]:
```suggestion
For this example, we'll install the `mysql-dev` service:
```

**Review by nunix — COMMENTED** [review]
Modified some sentences to active voice.

File: README.md (hunk 3 above)
Comment (mmartin24) [reply README.md]: thanks. applied

File: README.md (hunk 2 above)
Comment (mmartin24) [reply README.md]: thanks. applied

File: README.md (hunk 1 above)
Comment (mmartin24) [reply README.md]: thanks. applied

**Comment by mmartin24** [comment]
thanks @nunix for the suggestions. All of them are included

**Review by nunix — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -47,6 +47,31 @@ Wordpress needs a database to work. After visiting the route of your deployed ap
...
+#| mysql-dev      | 2022-09-22 12:45:05 +0200 CEST | 8.0.29  | A MySQL service that can be    |
+#|                |                                |         | used during development        |
+#| mongodb-dev    | 2022-09-22 12:45:05 +0200 CEST | 6.0.1   | A MongoDB service that can be  |
+#|                |                                |         | used during development        |
+#| redis-dev      | 2022-09-22 12:45:05 +0200 CEST | 6.2.7   | A Redis service that can be    |
+#|                |                                |         | used during development        |
+```
+More information about Epinio Service Catalog is available in [the docs](https://docs.epinio.io/references/customization/catalog).
+
+nunix59 minutes ago
```
Comment (enrichman) [inline README.md:73]: :sweat_smile:

**Pull request merged**